src/ui/pages/BridgeForm/QuoteList/QuoteList.tsx

```diff
@@ -0,0 +1,291 @@
+              transaction={quote.transaction}
+              configuration={configuration}
+            />
+          </React.Suspense>
+        ) : (
+          noValueDash
+        )}
+      </UIText>
+      <UIText kind="small/regular" style={{ textAlign: 'right' }}>
+        ~{formatSeconds(Number(quote.seconds_estimation))}
+      </UIText>
+    </HStack>
+  );
+}
+
+export function QuoteList({
+  configuration,
+  spendChain,
+  receiveAsset,
+  quotes,
+  selectedQuote,
+  onQuoteIdChange,
+  sortType,
+  onChangeSortType,
+}: {
+  configuration: CustomConfiguration;
+  spendChain: Chain;
+  receiveAsset: Asset;
+  quotes: Quote[];
+  selectedQuote: Quote | null;
+  onQuoteIdChange: (quoteId: string | null) => void;
+  sortType: QuoteSortType;
+  onChangeSortType: (sortType: QuoteSortType) => void;
+}) {
+  const formId = useId();
+
+  return (
+    <>
+      <DialogCloseButton style={{ position: 'absolute', top: 8, right: 8 }} />
+      <VStack gap={24} style={{ marginTop: 24 }}>
+        <SegmentedControlGroup childrenLayout="spread-children-evenly">
+          <SegmentedControlRadio
+            name="sortType"
+            value="amount"
+            checked={sortType === 'amount'}
+            onChange={() => onChangeSortType('amount')}
+          >
+            Max Received
+          </SegmentedControlRadio>
+          <SegmentedControlRadio
+            name="sortType"
+            value="time"
+            checked={sortType === 'time'}
+            onChange={() => onChangeSortType('time')}
+          >
+            Fastest Transfer
+          </SegmentedControlRadio>
+        </SegmentedControlGroup>
+        <VStack gap={16}>
+          <HStack
+            gap={4}
+            style={{
+              textAlign: 'center',
+              padding: '0 12px',
+              gridTemplateColumns: QUOTE_GRIP_TEMPLATE_COLUMNS,
+            }}
+          >
+            <div aria-hidden={true} />
+            <UIText kind="caption/accent" color="var(--neutral-600)">
+              Receive
+            </UIText>
+            <UIText kind="caption/accent" color="var(--neutral-600)">
+              Bridge Fee
+            </UIText>
+            <UIText kind="caption/accent" color="var(--neutral-600)">
+              Gas Fee
+            </UIText>
+            <UIText
+              kind="caption/accent"
+              color="var(--neutral-600)"
+              style={{ textAlign: 'right' }}
+            >
+              Time
+            </UIText>
+          </HStack>
+          <form
+            id={formId}
+            method="dialog"
+            onSubmit={(event) => {
+              event.stopPropagation();
+              const formData = new FormData(event.currentTarget);
+              const quoteId = formData.get('quoteId') as string | null;
+              if (quoteId || !selectedQuote) {
+                onQuoteIdChange(quoteId);
+              }
+            }}
+            style={{ maxHeight: 350, overflowY: 'auto' }}
```

Comment:
I think, we can use vh units here to let sidepanel users see all quotes without scroll

Reply:
Good idea, though currently it seems that the dialog grows with the list in sidepanel view